### PR TITLE
[bug] - Feed Recorded Future timeout while fetching indicators

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11172

## Description
image

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml [hunk 1]
```diff
@@ -155,6 +155,11 @@ configuration:
   name: risk_score_threshold
   required: false
   type: 0
+- additionalinfo: 'Improve the feed performance by removing the rawJSON from the indicators. Note: a loss of data may occure if checking this box'
```
Comment (ShirleyDenkberg) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml:158]:
```suggestion
- additionalinfo: 'Improve the feed performance by removing the rawJSON from the indicators. Note: A loss of data may occur if selecting this checkbox.'
```

File: Packs/FeedRecordedFuture/ReleaseNotes/1_1_3.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+#### Indicator Fields
+
+##### New: Recorded Future Risk Score
+
+- New: <~XSIAM> (Available from Cortex XSIAM 2.2).</~XSIAM>
```
Comment (ShirleyDenkberg) [inline Packs/FeedRecordedFuture/ReleaseNotes/1_1_3.md:6]:
```suggestion
New: <~XSIAM> (Available from Cortex XSIAM 2.2).</~XSIAM>
```

File: Packs/FeedRecordedFuture/ReleaseNotes/1_1_3.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
+
+#### Integrations
+
+##### Recorded Future RiskList Feed
+
+- Updated the Docker image to: *demisto/python3:3.11.9.103066*.
+- Added the parameter **Improve performance**
```
Comment (ShirleyDenkberg) [inline Packs/FeedRecordedFuture/ReleaseNotes/1_1_3.md:14]:
```suggestion
- Added the **Improve performance** parameter.
- Updated the Docker image to: *demisto/python3:3.11.9.103066*.
```

**Comment by ShirleyDenkberg** [comment]
@RotemAmit @Ni-Knight Doc review completed.

File: Packs/FeedRecordedFuture/IndicatorFields/incidentfield-recordedfutureriskscore.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (Ni-Knight) [inline Packs/FeedRecordedFuture/IndicatorFields/incidentfield-recordedfutureriskscore.json:1]: Please rename as `indicatorfield-XXXXX`

File: Packs/FeedRecordedFuture/IndicatorFields/incidentfield-recordedfutureriskscore.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    "id": "indicator_recordedfutureriskscore",
+    "version": -1,
+    "modified": "2024-07-17T12:28:16.838935726Z",
+    "name": "Recorded Future Risk Score",
+    "ownerOnly": false,
+    "cliName": "recordedfutureriskscore",
+    "type": "number",
```
Comment (Ni-Knight) [inline Packs/FeedRecordedFuture/IndicatorFields/incidentfield-recordedfutureriskscore.json:8]: Do we want a number or a string here?

**Review by Ni-Knight — APPROVED** [review]
Please see the rename comment, other than that seems fine :)

**Review by RotemAmit — COMMENTED** [review]
Nice!
Just fix test_stream_compressed_data_file_content, it fails in pre-commit.

File: Packs/FeedRecordedFuture/IndicatorFields/incidentfield-recordedfutureriskscore.json (hunk 5 above)
Comment (MLainer1) [reply Packs/FeedRecordedFuture/IndicatorFields/incidentfield-recordedfutureriskscore.json:8]: Well, it's up to you, can be either way

File: Packs/FeedRecordedFuture/ReleaseNotes/1_1_3.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+##### New: Recorded Future Risk Score
+
+New: <~XSIAM> (Available from Cortex XSIAM 2.2).</~XSIAM>
+<~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
+
+#### Integrations
+
+##### Recorded Future RiskList Feed
+
+- Added the **Remove rawJSON from indicators** parameter.
```
Comment (dantavori) [inline Packs/FeedRecordedFuture/ReleaseNotes/1_1_3.md:13]:
```suggestion
- Added the *Remove rawJSON from indicators* parameter.
```
I'm missing a note about the goal of this parameter

**Review by dantavori — APPROVED** [review]
nice

**Pull request merged**